GitExtUtils/MruCache.cs

```diff
@@ -62,6 +65,8 @@ public void Add(TKey key, TValue value)
                 if (_entries.Count == Capacity)
                 {
                     var last = _entries.Last;
+                    Assumes.NotNull(last);
```

Comment:
`Validates.NotNull`. I suggest searching for usages of `Assumes.NotNull` and replacing them.